packages/workflow/src/utils.ts

```diff
@@ -10,23 +16,26 @@ export const deepCopy = <T>(source: T, hash = new WeakMap(), path = ''): T => {
 	if (hash.has(source)) {
 		return hash.get(source);
 	}
-	// Date
-	if (source instanceof Date) {
```

Comment:
cloning dates was a great addition. but unfortunately it wasn't backward compatible. 
If we decide to add this code back, we should update the rest of the app code to also handle cloned date objects instead of string.